webpack.requirejs_jquery_q.config.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
```

Comment:
Comments: added
Testing: it works (with the upcoming null handling change), but we have to map `q` to `vendor/lib/q` in the widget
Size diff: old - 46.65 kB, new - 46.30 kB